packages/components/src/components/accordion/_accordion.scss

```diff
@@ -100,11 +100,12 @@
 
   .#{$prefix}--accordion__item--disabled,
   .#{$prefix}--accordion__item--disabled + .#{$prefix}--accordion__item {
-    border-top: 1px solid $disabled-02;
+    border-top: 1px solid $button-disabled;
+    //should this be $border-disabled (that would make it disabled01)
   }
 
   li.#{$prefix}--accordion__item--disabled:last-of-type {
-    border-bottom: 1px solid $disabled-02;
+    border-bottom: 1px solid $button-disabled;
```

Comment:
`button-disabled` is actually darker than the `border-subtle` color (in light themes). Disabled borders need to has less contrast. `$border-disabled` is correct.

Reply:
Yeah, I was confused bc the existing styles use `disabled-02` and that doesn't match up to `border-disabled`. Wasn't sure if it was a style bug or intentional.

Reply:
Yeah it can get confusing. I had to really think about this one for a minute.